main/src/ca/uwaterloo/flix/language/ast/Ast.scala

```diff
@@ -564,7 +568,7 @@ object Ast {
 
       case object Do extends Expr
 
-      case class InvokeMethod(tpe: ca.uwaterloo.flix.language.ast.Type, name: Name.Ident) extends Expr
+      case class InvokeMethod(tpe: ca.uwaterloo.flix.language.ast.Type, name: ast.Name.Ident) extends Expr
```

Comment:
What happened here?

Reply:
Strange name conflict. In this file, there's a `Name` object and now `SyntacticContext.Name`. That line is referring to the former, but since it's in the `SyntacticContext` object, the latter shadows the former. So to refer to the outer `Name` object, I'm importing ast and then doing `ast.Name`. It's a little clunky

Reply:
I see.